fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -304,7 +304,7 @@ valueExpression
 datetimeUnit
     : YEAR | MONTH
     | WEEK | DAY
-    | HOUR | MINUTE | SECOND
+    | HOUR | MINUTE | SECOND | MICROSECOND
```

Comment:
do not add this, if u do not want to implement dateAdd / dateSub / timestampDiff function in MICROSECOND unit